spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -82,6 +82,68 @@ The xref:getting-started#getting-started[Getting Started] section shows you how
 
 Note that the shell calls the Data Flow Servers' REST API. For more information on making HTTP requests directly to the server, consult the <<api-guide, REST API Guide>>.
 
+=== Stream Application DSL
+
+The Stream Pipeline DSL described in the previous section automatically sets the input and output binding properties of each Spring Cloud Stream application.
+This can be done because there is only one input and/or output destination in a Spring Cloud Stream application that uses the provided binding interface of a `Source`, `Processor`, or `Sink`.
+However, a Spring Cloud Stream application can define a custom binding interface such as the one shown below
```

Comment:
Change "one shown below" to "following". (The modern convention is to avoid "above" and "below" because we can't control where the page breaks land, so content may end up above the reference but on the next page.) Also, please add a colon. So it should end with "following:"